### PR TITLE
update polkadot&cumulus

### PR DESCRIPTION
**Pull request by weichweich**
* Updates Substrate and Cumulus to the latest rococo-v1 
* Required to reconnect to Rococo after reset on 2021-02-11

### CODE REVIEW

File: nodes/parachain/src/service.rs [hunk 1]
```diff
@@ -14,16 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-// If you feel like getting in touch with us, you can do so at info@botlabs.org
```
Comment (wischli) [inline nodes/parachain/src/service.rs:17]: I suppose this was removed by mistake?

File: scripts/update-spec.py [hunk 2]
```diff
@@ -250,14 +259,13 @@ def setup_spec(authorities, session_keys, endowed, root, outpath: pathlib.Path,
                 val_3_pub: DEFAULT_MONEY,
             },
             val_1_pub,
-            pathlib.Path.cwd() / "dev-specs/kilt-parachain/relay-stage.json",
+            chain_file=args.roc_staging_relay,
             extras={
-                "name": "KILT Staging Relaychain",
-                "id": "kilt_staging_relay",
-                "chainType": "Live",
+                # changing those breaks the chainspec building. :(
```
Comment (wischli) [inline scripts/update-spec.py:264]: 😢

**Review by wischli — CHANGES_REQUESTED** [review]

File: nodes/parachain/src/service.rs (hunk 1 above)
Comment (weichweich) [reply nodes/parachain/src/service.rs:17]: oh yes

Comment (weichweich) [reply nodes/parachain/src/service.rs:17]: fixed in 7289b68

**Review by wischli — APPROVED** [review]
LGTM

**Pull request merged**